extension/src/CSV/ExportXliffTSV.ts

```diff
@@ -0,0 +1,44 @@
+import { isNullOrUndefined } from "util";
+import { CustomNoteType, Xliff } from "../XLIFFDocument";
+import { CSV } from "./CSV";
+
+
+export function createXliffTSV(xlf: Xliff): CSV {
+    let csv = new CSV();
+    csv.headers =
+        [
+            "Id",
+            "Source",
+            "Target",
+            "Developer Note",
+            "Max Length",
+            "Comment",
+            "Xliff Generator Note",
+            CustomNoteType.RefreshXlfHint,
+            "State",
+            "State Qualifier"
+        ];
+    xlf.transunit.forEach(tu => {
+        csv.addLine([
+            tu.id,
+            tu.source,
+            tu.targetTextContent,
+            tu.getNoteTextContent("Developer"),
+            isNullOrUndefined(tu.maxwidth) ? "" : tu.maxwidth.toString(),
+            "", // comment
+            tu.getNoteTextContent("Xliff Generator"),
```

Comment:
Update from upstream/master and you'll have this one:
```suggestion
            tu.xliffGeneratorNote().textContent,
```

Reply:
Bah, I prepped that last night but forgot to push :D